### PR TITLE
Mali/fix remove existing ads

### PR DESCRIPTION
**Pull request by maliheh-deriv**
# Description : 
The function "removeExistingAds" should remove all ads but it only remove the first ad. This PR fixed the issue.

### CODE REVIEW

File: cypress/support/commands/p2p.js [hunk 1]
```diff
@@ -513,16 +513,27 @@ Cypress.Commands.add('c_postAd', () => {
...
+            .should('be.enabled')
+            .click()
+            .should('not.exist', {
+              timeout: 10000,
+            })
+          if (index < numberOfAds - 1) {
+            cy.wait(1000)
+          }
+        }
+      )
```
Comment (bilalmajid-deriv) [inline cypress/support/commands/p2p.js:536]: can you please add logs for index and numberOfAds so that it can help in debugging later on thanks

**Review by bilalmajid-deriv — APPROVED** [review]
LGTM

File: cypress/support/commands/p2p.js [hunk 2]
```diff
@@ -513,16 +513,29 @@ Cypress.Commands.add('c_postAd', () => {
...
+            'be.visible'
+          )
+          cy.findByRole('button', { name: 'Delete' })
+            .should('be.enabled')
+            .click()
+            .should('not.exist', {
+              timeout: 10000,
+            })
+          if (index < numberOfAds - 1) {
+            cy.wait(1000)
```
Comment (meeyun-deriv) [inline cypress/support/commands/p2p.js:535]: why this wait is needed?

**Review by meeyun-deriv — COMMENTED** [review]
From our conversation, i thought we are changing the solution to create fresh account on the go. Can you please confirm if this is still the case?

File: cypress/support/commands/p2p.js (hunk 2 above)
Comment (maliheh-deriv) [reply cypress/support/commands/p2p.js:535]: I have added this wait to provide a buffer after deleting each row, which helps avoid flaky failures.

**Review by meeyun-deriv — APPROVED** [review]
LGTM

**Pull request merged**